### DISCUSSION TITLE
记一次 WiFi 断开链接

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#433] [Announcements]
# 记一次 WiFi 断开链接

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2019/01/04/record-for-wifi-disconnect/

**Comment by jixingyou** [comment]
hihi，请问下，后来这个问题有定位到原因吗？

**Reply by Xuanwo** [reply]
没有，后来换设备了，再后来不在那上班了 :laughing:

**Reply by jixingyou** [reply]
哦哦好的，还是谢谢老哥
